devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -275,25 +301,22 @@ private refreshAll(includeMeterGet = true) {
 	sendHubCommand refresh(endpoints,includeMeterGet)
 }
 
-def childRefresh(deviceNetworkId, includeMeterGet = true) {
+def childRefresh(deviceNetworkId, includeMeterGet = deviceIncludesMeter()) {
 	def switchId = getSwitchId(deviceNetworkId)
 	if (switchId != null) {
 		sendHubCommand refresh([switchId],includeMeterGet)
 	}
 }
 
 def refresh(endpoints = [1], includeMeterGet = true) {
```

Comment:
I've just figured out, that you also should change `includeMeterGet = true` to `includeMeterGet = deviceIncludesMeter()`.

Reply:
Ok done.